tests/test_pytest_cov.py

```diff
@@ -100,6 +100,12 @@ def test_run_target():
     p.join()
 '''
 
+try:
+    import xdist
+    HAS_XDIST = True
+except:
```

Comment:
`except ImportError`

Reply:
`requires_xdist = pytest.mark.skipif('xdist' in sys.modules, reason='pytest xdist is not active')`